client/scss/components/_numbered_list.scss

```diff
@@ -148,6 +148,22 @@ $numbered-list-number-size-large: 96px;
     border-width: 0 1px 0 0;
   }
 
+  .numbered-list__item--turquoise {
+    border-bottom: $v-spacing-unit solid color('turquoise');
+  }
+
+  .numbered-list__item--orange {
+    border-bottom: $v-spacing-unit  solid color('orange');
```

Comment:
You've got two spaces after `$v-spacing-unit` on these three properties – it's making my eyes bleed!